intl_tel_widget/views/assets.xml

```diff
@@ -15,4 +15,14 @@
         </xpath>
     </template>
 
+    <template id="assets_frontend" name="intl tel widget frontend assets" inherit_id="web.assets_frontend">
+        <xpath expr="." position="inside">
+
+            <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intl-tel-input@17.0.3/build/css/intlTelInput.css"/>
```

Comment:
What do you think about using `/intl_tel_widget/static/lib/intl-tel-input/css/intlTelInput.css` instead of loading it from an external site?

Reply:
What do you think about using  `/intl_tel_widget/static/lib/intl-tel-input/js/intlTelInput-jquery.js` instead of loading it from an external site?